tests/tools/test_klayout.py

```
@@ -311,10 +311,29 @@ def test_img2stream():
     assert proj.run()
 
     gds = proj.find_result("gds", step="image")
+    lef = proj.find_result("lef", step="image")
+    assert os.path.isfile(gds)
+    assert os.path.isfile(lef)
 
     with open(gds, 'rb') as gds_file:
         data = gds_file.read()
-        assert hashlib.md5(data).hexdigest() == "98e5472b73430afa2952ec8393cdef2a"
+        assert hashlib.md5(data).hexdigest() == "0c8a1f81af4a731ecb4f86f3f4bac591"
```

Ruff (0.15.15)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)